src/toast/toast.tsx

```diff
@@ -68,64 +77,47 @@ export const Toast: React.FC = ({
     });
 
     // =============================================================================
-    // EFFECTS
-    // =============================================================================
-    useEffect(() => {
-        setVisible(true);
-    }, []);
-
-    useEffect(() => {
-        if (!autoDismiss) return;
-
-        let timer = 4;
-
-        const interval = setInterval(() => {
-            timer--;
-            if (timer === 0) {
-                clearInterval(interval);
-                setVisible(false);
-            }
-        }, 1000);
-
-        return () => clearInterval(interval);
-    }, [autoDismiss]);
-
-    // =============================================================================
-    // EVENT HANDLERS
+    // RENDER FUNCTIONS
     // =============================================================================
-    const handleDismiss = () => {
-        setVisible(false);
+    const renderIcon = () => {
+        switch (type) {
+            case "success":
+                return <TickCircleFillIcon />;
+            case "warning":
+                return <ExclamationTriangleFillIcon />;
+            case "error":
+                return <ExclamationCircleFillIcon />;
+            case "info":
+                return <ICircleFillIcon />;
+            default:
+                return null;
+        }
     };
 
     return (
-        <animated.div>
-            <Wrapper
-                style={transitions}
-                $type={type}
-                $autoDismiss={autoDismiss}
-                {...otherProps}
-            >
-                <IconContainer>{renderIcon()}</IconContainer>
-                <TextContainer>
-                    {title && (
-                        <Title $type={type} weight={"semibold"}>
-                            {title}
-                        </Title>
-                    )}
-                    {description && (
-                        <Description $type={type}>
-                            {!title ? (
-                                <Text.Body>{description}</Text.Body>
-                            ) : (
-                                <Text.BodySmall>{description}</Text.BodySmall>
-                            )}
-                        </Description>
-                    )}
-                </TextContainer>
-                <DismissButton $type={type} onClick={handleDismiss}>
-                    <CloseIcon />
-                </DismissButton>
-            </Wrapper>
-        </animated.div>
+        <Wrapper style={transitions} $type={type} {...otherProps}>
+            {renderIcon()}
+            <TextContainer>
+                {label && (
+                    <Title $type={type} weight="semibold">
+                        {label}
+                    </Title>
+                )}
+                {secondaryLabel && (
+                    <Description $type={type}>
+                        {!label ? (
+                            <Text.Body>{secondaryLabel}</Text.Body>
```

Comment:
Actually this is incorrect. My earlier understanding was `label` is still the main input. Only if you specified `secondaryLabel` then the `label` will be bolded

Reply:
Actually the expected result is the description is mandatory and the title is optional? should i revert to use `label` as description and `secondaryLabel` as title?